python_modules/dagster/dagster/_daemon/daemon.py

```diff
@@ -296,3 +297,15 @@ def run_iteration(
         workspace_process_context: IWorkspaceProcessContext,
     ) -> DaemonIterator:
         yield from execute_monitoring_iteration(workspace_process_context, self._logger)
+
+
+class ConcurrencySlotsDaemon(IntervalDaemon):
+    @classmethod
+    def daemon_type(cls) -> str:
+        return "CONCURRENCY_SLOTS"
+
+    def run_iteration(
+        self,
+        workspace_process_context: IWorkspaceProcessContext,
+    ) -> DaemonIterator:
+        yield from execute_concurrency_slots_iteration(workspace_process_context, self._logger)
```

Comment:
should we just have the run monitoring daemon take care of this since thats where the setting is?